### ISSUE TITLE
Search results limited to 500 entries?

### ISSUE THREAD

**Issue by CTegtmeyer** [spawaskar-cora/cora-docs#325]
This is more a question/comment than an issue. Can we expand the number to entries the search results are limited to? We have accessions that are narrowed down all the way to Provenance 2 that still return over 500 results when searched. I don't know if 500 was chosen for the capabilities of the search function and if it cannot be expanded, but I thought I'd mention it.

**Comment by CTegtmeyer** [comment]
I wanted to bring this issue up again, because I am running into problems with this. Because the search is limited to 500 elements, if you search for an element that does not come up in the initial 500, it says the element does not exist within the accession, thereby basically preventing you from accessing the through a search by accession. This is an issue if you need to go back to an element to add or edit information (say when you're waiting for a bone group to populate to add articulation information). Has any progress been made to expand this limit?

**Comment by SachinPawaskarUNO** [comment]
@CTegtmeyer I agree that this limit needs to be addressed. However I do not agree that this element cannot be accessed. There are other searches that you should be able to do that should give you a smaller subset than the current 500 limit, such as composite key, designator, etc.

**Comment by CTegtmeyer** [comment]
this issue has been resolved
